src/lib/seen-this-session.js

```diff
@@ -0,0 +1,26 @@
+import auth from 'panoptes-client/lib/auth';
+
+const subjectsSeenThisSession = [];
+
+auth.listen('change', () => {
+  return subjectsSeenThisSession.splice(0);
+});
+
+function addAlreadySeen(workflowId, subjectIds) {
+  subjectIds.map((subjectId) => {
+    subjectsSeenThisSession.push(`${workflowId}/${subjectId}`);
+  });
+}
+
+function checkAlreadySeen(workflow, subject) {
+  return subjectsSeenThisSession.includes(`${workflow.id}/${subject.id}`);
+}
+
+if (window) {
+  window.subjectsSeenThisSession = subjectsSeenThisSession;
```

Comment:
This works, but why are we saving the array of Subjects seen (in one session) into the global namespace instead of chucking said array into a duck? I'm not opposed to using global variables, but I'd like to know why we're breaking with our usual data store pattern.

Reply:
No reason really, this was just how it was done in PFE. For our use, it probably would make more sense to put it in a duck, which I can get done in another commit.